R/plots-qc.R

```diff
@@ -113,18 +120,32 @@ qc_variance_hist <- function(gimap_dataset, wide_ar = 0.75){
 #' @examples \dontrun{
 #' gimap_dataset <- get_example_data("gimap")
 #' qc_constructs_countzero_bar(gimap_dataset)
+#' 
+#' #or if you want to select a specific column(s) for looking at where/which samples zero counts are present for
+#' qc_constructs_countzero_bar(gimap_dataset, filter_zerocount_target_col = c(3:5))
+#' 
+#' #or if you want to select a specific column(s) for the final day/sample replicates
+#' qc_constructs_countzero_bar(gimap_dataset, filter_replicates_target_col = c(3:5))
+#' 
+#' #or some combination of those
+#' qc_constructs_countzero_bar(gimap_dataset, filter_zerocount_target_col = c(3:5), filter_replicates_target_col = c(3:5))
 #' }
 #'
 
-qc_constructs_countzero_bar <- function(gimap_dataset, wide_ar = 0.75){
+qc_constructs_countzero_bar <- function(gimap_dataset, filter_zerocount_target_col = NULL, filter_replicates_target_col = NULL, wide_ar = 0.75){
 
-  qc_filter_output <- qc_filter_zerocounts(gimap_dataset)
+  if(is.null(filter_zerocount_target_col)){filter_zerocount_target_col <- c(1:ncol(gimap_dataset$raw_counts))}
+  
+  qc_filter_output <- qc_filter_zerocounts(gimap_dataset, filter_zerocount_target_col = filter_zerocount_target_col)
+  
+  if(is.null(filter_replicates_target_col)){ filter_replicates_target_col <- c((ncol(gimap_dataset$transformed_data$log2_cpm)-2) : ncol(gimap_dataset$transformed_data$log2_cpm))} #last 3 columns of the data
+  
 
   return(
-    example_counts[qc_filter_output$filter, c(3:5)] %>%
+    example_counts[qc_filter_output$filter, filter_replicates_target_col] %>%
```

Comment:
```suggestion
    gimap_dataset$raw_counts[qc_filter_output$filter, filter_replicates_target_col] %>%
```
I think this is what you mean?

Reply:
Thanks for catching that!